src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -24,6 +24,193 @@ const formConfig = {
   title: 'Apply for education benefits',
   subTitle: 'Form 22-1990',
   chapters: {
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        benefitEligibility: {
+          title: 'Veteran information',
+          path: 'veteran-information',
+          uiSchema: {
+          },
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        }
+      }
+    },
+    benefitsEligibility: {
+      title: 'Benefits Eligibility',
+      pages: {
+        benefitEligibility: {
+          title: 'Benefits eligibility',
+          path: 'benefits-eligibility/benefits-selection',
+          uiSchema: {
+          },
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        },
+        benefitRelinquishment: {
+          title: 'Benefits relinquishment',
+          path: 'benefits-eligibility/benefits-relinquishment',
+          depends: {
+            chapter33: true
+          },
+          uiSchema: {
+          },
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        }
+      }
+    },
+    militaryHistory: {
+      title: 'Military History',
+      pages: {
+        servicePeriods: {
+          title: 'Service periods',
+          path: 'military-history/service-periods',
+          uiSchema: {
+          },
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        },
+        militaryService: {
+          title: 'Military service',
+          path: 'military-history/military-service',
+          uiSchema: {
+          },
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        },
+        rotcHistory: {
+          title: 'ROTC history',
+          path: 'military-history/rotc-history',
+          uiSchema: {
+          },
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        },
+        contributions: {
+          title: 'Contributions',
+          path: 'military-history/contributions',
+          uiSchema: {
+          },
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        }
+      }
+    },
+    educationhistory: {
+      title: 'Education History',
+      pages: {
+        educationHistory: {
+          title: 'Education history',
+          // There's only one page in this chapter (right?), so this url seems a
+          //  bit heavy-handed.
+          path: 'education-history/education-information',
```

Comment:
Should we make this consistent with the current 1990 routes or take this opportunity to make the 1990 routes consistent with the other education forms (e.g. `path: 'education/history'`)?